docs/modules/configuring/con-configuring-kafkaservice-cipher.adoc

```diff
@@ -0,0 +1,31 @@
+[id='con-configuring-kafkaservice-cipher-{context}']
+= Controlling the TLS ciphers used
+
+Some cipher suites, which are enabled by default for interoperability purposes, are no longer consider secure.
```

Comment:
Perhaps this needs more context?

_Some older cipher suites enabled by default for TLS connections between clients and virtual clusters are now considered insecure._